ext/include/opentelemetry/ext/http/client/curl/http_operation_curl.h

```diff
@@ -267,7 +268,7 @@ class HttpOperation
     return SetCurlPtrOption(option, list);
   }
 
-  CURLcode SetCurlLongOption(CURLoption option, long value);
+  CURLcode SetCurlLongOption(CURLoption option, int64_t value);
```

Comment:
I think this will break curl on 32bits platforms.

Curl does use `long` in the API, not int64_t.

Please revert.

Reply:
Noted and reverted.